environment/src/test/java/com/sequenceiq/environment/environment/dto/EnvironmentDtoConverterTest.java

```diff
@@ -0,0 +1,72 @@
+package com.sequenceiq.environment.environment.dto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
+import com.sequenceiq.cloudbreak.auth.security.CrnUserDetailsService;
+import com.sequenceiq.cloudbreak.tag.CostTagging;
+import com.sequenceiq.environment.environment.domain.Environment;
+import com.sequenceiq.environment.environment.domain.EnvironmentAuthentication;
+import com.sequenceiq.environment.environment.validation.validators.PublicKeyValidator;
+import com.sequenceiq.environment.tags.service.AccountTagService;
+import com.sequenceiq.environment.tags.service.DefaultInternalAccountTagService;
+import com.sequenceiq.environment.tags.v1.converter.AccountTagToAccountTagResponsesConverter;
+
+class EnvironmentDtoConverterTest {
+
+    private final PublicKeyValidator publicKeyValidator = Mockito.mock(PublicKeyValidator.class);
```

Comment:
Why not use `@Mock` on instance fields for automatic mock creation and `@ExtendWith(MockitoExtension.class)` on the test class?

Reply:
I prefer to use @Mock, but it seemed to me that in environment service this is the preferred way. I can change it if you like.